### ISSUE TITLE
Evaluation

### ISSUE THREAD

**Issue by lethenam** [bendanpuppy/Free3R-GS#2]
Could you clarify how the evaluation was conducted?

Most of the compared methods are designed for datasets with known camera intrinsics and extrinsics. In your setting, did you first estimate the intrinsic and extrinsic parameters for all images and then use these estimated parameters to train and evaluate SOTA methods? 

In addition, the paper appears to highlight the use of EndoDAC as the depth estimation model. However, it is unclear why the results obtained using EndoDAC are presented as part of your method in the comparison tables. 

Furthermore, the paper provides very limited details about the networks used for intrinsic and extrinsic estimation. Can you provide more information regarding their architecture, training procedure, and accuracy? and pretrained weights?
